pkg/tenant/db/members.go

```diff
@@ -104,6 +120,11 @@ func CreateMember(ctx context.Context, member *Member) (err error) {
 
 	member.Created = time.Now()
 	member.Modified = member.Created
+	member.DateAdded = member.Created
+	member.LastActivity = member.Created
+
+	// TODO: Add check to display correct user status.
+	member.Status = MemberConfirmed
```

Comment:
Since `member` is required for this method, I think we should leave the status up to the caller and rely on validation in this method to make sure the status field is set before saving to the database.

Reply:
Makes sense. I'll add the change.